generators/app/templates/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "<%=projectName%>",
+  "version": "0.1.0",
+  "description": "<%=projectName%>",
+  "main": "lib/<%=projectClassName%>.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/<%=projectName%>.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "<%=projectName%>",
+    "<%=projectClassName%>",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/<%=projectName%>/issues"
+  },
+  "homepage": "https://github.com/cerner/<%=projectName%>#readme",
+  "scripts": {
+    "build": "babel src --out-dir lib",
+    "lint": "npm run lint-js && npm run lint-scss",
+    "lint-js": "eslint --ext .jsx --ext .js .",
+    "lint-scss": "stylelint src/**/*.scss",
+    "posttest": "npm run lint",
```

Comment:
Are we ok with these posttest and pretest scripts rather than doing some kind of combined test or verify script?